IRONIC.md

```diff
@@ -20,3 +20,16 @@ inside of `` user_osa_variables_overrides ``.
 
 The IP addresses referenced should be the same one referenced in
 `` extra_lb_vip_addresses `` that have been setup during installation.
+
+### Configure neutron DHCP to be non-authoritative
+
+To prevent us from having to recreate the DHCP server, neutron needs to be set
```

Comment:
It's not to prevent the recreation of a dhcp server, but to prevent conflicts between them.